plugin.xml

```diff
@@ -99,6 +99,8 @@
     <!-- ios -->
     <platform name="ios">
         <preference name="API_KEY_FOR_IOS" />
+        <preference name="LOCATION_WHEN_IN_USE_DESCRIPTION" default="Show your location on the map"/>
+        <preference name="LOCATION_ALWAYS_USAGE_DESCRIPTION" default="Trace your location on the map" />
```

Comment:
Do these strings ever show up anywhere? We definitely do not want these strings showing up anywhere